bower.json

```diff
@@ -32,6 +32,7 @@
         "requirejs-plugins": "~1.0.2",
         "At.js": "emord/At.js#override-insert",
         "Caret.js": "0.2.2",
+        "fuse.js": "1.2.2",
```

Comment:
Would it make sense to pin this at `"~1.2.2"`? I believe that tells bower that the original version we used was 1.2.2, but upgrade to newer versions of 1.2 if they become available. This is great if the library uses semantic versions; not so great if not.